p2p/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.9] - 2022-12-04
+
+- Raintree mempool cannot grow unbounded anymore. It's now bounded by a constant limit and when new nonces are inserted the oldest ones are removed.
```

Comment:
1. No it's not but it can be. Stay tuned

2. One of my goals in life is to make complex things very easy for myself and potentially anyone interested :)

No, jokes apart, you are right but it's not LIFO but FIFO

If the cap/limit on the size of the set is 2:

| T | What's up?| State |
| -- | -- | -- |
| 0 | put(first) | [ ] |
| 1 | put(second) | [ first ] | 
| 2 | put(third) | [ first, second ] | 
| 3 | 🅰️ where's **FIRST**? I saw it going **IN** at T 0 <br > 🅱️ dude, **FIRST** is **OUT**, it had to make space for the new guy, "three" | [ second, third ] |

Reply:
1. is now implemented in https://github.com/pokt-network/pocket/pull/374/commits/4ce80bc24b0cbad8a27df6d7468475626954defb